### ISSUE TITLE
Add method to unlink link_selections

### ISSUE THREAD

**Issue by philippjfr** [holoviz/holoviews#4563]
Currently it's easy to add new elements to a linked selection by making an instance and calling it on new elements. However there is no easy way to unlink an element once it has been added.

**Comment by jonmmease** [comment]
Do you have in mind what this would mean?  Are you picturing trying to fully invert the `link_selections` transformation, or keeping the element selectable, but severing it from being linked to other elements?

The former case doesn't feel well defined at first thought. But I think the latter case is something that we could reason about on top of https://github.com/holoviz/holoviews/pull/4572.

**Comment by philippjfr** [comment]
Right, unlinking it from the other elements is all I was imagining here. I had this working but suspect it'll need reworking after your PR was merged. My solution also wasn't all that pretty, basically I had a weakref mapping from the object you passed as input to `link_selections` to all the objects that then have to get unlinked.
